code/pizzabot.js

```diff
@@ -1,7 +1,55 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const vegetarian = 'Vegetarian Pizza'
+const hawaiian = 'Hawaiian Pizza'
+const pepperoni = 'Pepperoni Pizza'
 
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+alert(`Hello there, we are happy to serve you pizza! On todays menu we have ${vegetarian}, ${hawaiian} and ${pepperoni}!`);
+const orderName = prompt('Enter the name of the pizza you want to order today!');
+
+/* This function will check if the order is correct*/
+const checkOrderName = (orderName) => {
+  if (orderName == hawaiian || orderName == vegetarian || orderName == pepperoni) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+/*Invoke if orderName exist*/
+const order = checkOrderName(orderName);
+
+
+/*total cost function*/
+const totalCost = (orderQuantity) => {
+  orderTotal = (orderQuantity * pizzaPrice);
+  return orderTotal;
+}
+
+/* Cooking time function */
+const cookingTime = (orderQuantity) => {
+  if (orderQuantity < 3) {
+    return (10);
+  } else if (orderQuantity < 6) {
+    return (15);
+  } else {
+    return (20);
+  }
+}
+
+
+const checkOrder = (order) => {
+  if (order === true) {
```

Comment:
This means `true` actually! And `!order` means false! 
```suggestion
  if (order) {
```